modules/ui/CaptionUI.py

```diff
@@ -275,9 +284,10 @@ def refresh_image(self):
             if self.display_only_mask:
                 self.image.configure(light_image=resized_pil_mask, size=resized_pil_mask.size)
             else:
+                norm_min = 0.2
```

Comment:
Great idea with the normalization, but this doesn't work as I would expect.
1. When I create a fresh mask with a single line of 0.5 alpha, shouldn't that be displayed with full brightness?
2. Creating a fully white mask throws an exception
3. The darkest color (0.2) is a bit too dark in my opinion. 0.4 worked pretty well, because you could still see some detail of the original image in the dark areas. 0.3 should be a good compromise.

Also: this is a pretty performance critical function, so calling np_mask.min() twice isn't a great idea. I think you should call it only once and save the result in a variable.

Reply:
Thank you for all the feedback.  I'll work on it soon.  

With regards to the normalization, I would expect a 1.0 alpha to be the only thing to be full brightness.   My idea was to basically compress the range 0 - 1 into a (with your recommendation for the darkness) 0.4 - 1 instead.  This would mean the brightest would be 1 and 50% would be 0.7 which is halfway between 0.4 and 1.

Again thank you for the thorough review I will fix all of the requested items

Reply:
Fully white mask bug... where the min is 1.0 and a divide by 0 happens --- smh rookie mistake

Reply:
1. did not know if the normalization should be changed, and I left the algorithm as is.  For what it's worth, if changing the normalization from being from "min - 1" over to "min - max" (the behavior you mentioned) is just a two line change -- the incorporation of np_mask.max() instead of a hard coded 1.0
2. fixed. great catch
3. fixed. not a problem